Extension/MultipathUpgrade/Model/AvailableResources.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.CampaignSystem.ViewModelCollection;
+using TaleWorlds.Core;
+
+namespace YAPO.MultipathUpgrade.Model
+{
+    public class AvailableResources
+    {
+        public AvailableResources(bool hotkeyControl)
+        {
+            int cost = 0;
+            if (!hotkeyControl)
+            {
+                int dailyCost = GetMostAccurateDailyCost();
+                cost = dailyCost * YapoSettings.Instance.DaysToPayDailyCostsBuffer;
+            }
+
+            AvailableGold = Hero.MainHero.Clan.Gold + cost;
```

Comment:
Shouldn't the cost be subtracted from the available gold?
```suggestion
            AvailableGold = Hero.MainHero.Clan.Gold - cost;
```

Reply:
Yeah, but the cost is -x and y - -x ads it up